src/layouts/Footer.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
```

Comment:
in React v17+ you don't need to `import React` anymore

Reply:
Removed imports.